dags/bqetl_org_mozilla_fenix_derived.py

```diff
@@ -0,0 +1,50 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "amiyaguchi@mozilla.com",
+    "start_date": datetime.datetime(2020, 10, 18, 0, 0),
+    "email": ["amiyaguchi@mozilla.com", "telemetry-alerts@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+with DAG(
+    "bqetl_org_mozilla_fenix_derived",
+    default_args=default_args,
+    schedule_interval="@daily",
+) as dag:
+
+    org_mozilla_fenix_derived__geckoview_version__v1 = bigquery_etl_query(
+        task_id="org_mozilla_fenix_derived__geckoview_version__v1",
+        destination_table="geckoview_version_v1",
+        dataset_id="org_mozilla_fenix_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="amiyaguchi@mozilla.com",
+        email=["amiyaguchi@mozilla.com", "telemetry-alerts@mozilla.com"],
+        date_partition_parameter="submission_date",
```

Comment:
I don't think `date_partition_parameter` should be here since it isn't incremental.  Is `incremental: false` expected to take this out?

Reply:
`date_partition_parameter` is the suffix to the destination table e.g. `geckoview_version_v1$20201018`. The incremental keyword applies to `depends_on_past` in the airflow parameters.

Reply:
Ok I thought this wasn't a partitioned table

Reply:
I thought the reason you defined a submission_date parameter was because you didn't want `date_partition_parameter` defined